code/physics/physics.cpp

```diff
@@ -162,47 +162,65 @@ void physics_sim_rot(matrix * orient, physics_info * pi, float sim_time )
 	Assert(is_valid_vec(&pi->desired_rotvel));
 
 	// Handle special case of shockwave
-	shock_amplitude = 0.0f;
-	if ( pi->flags & PF_IN_SHOCKWAVE ) {
-		if ( timestamp_elapsed(pi->shockwave_decay) ) {
+	if (pi->flags & PF_IN_SHOCKWAVE) {
+		if (timestamp_elapsed(pi->shockwave_decay)) {
 			pi->flags &= ~PF_IN_SHOCKWAVE;
 			rotdamp = pi->rotdamp;
-		} else {
- 			shock_fraction_time_left = timestamp_until( pi->shockwave_decay ) / (float) SW_BLAST_DURATION;
+		}
+		else {
+			shock_fraction_time_left = timestamp_until(pi->shockwave_decay) / (float)SW_BLAST_DURATION;
 			rotdamp = pi->rotdamp + pi->rotdamp * (SW_ROT_FACTOR - 1) * shock_fraction_time_left;
 			shock_amplitude = pi->shockwave_shake_amp * shock_fraction_time_left;
 		}
-	} else if ( pi->flags & PF_NO_DAMP ) {
+	}
+	else if (pi->flags & PF_NO_DAMP) {
 		rotdamp = 0.0f;
-	} else {
+	}
+	else {
 		rotdamp = pi->rotdamp;
 	}
 
-	// Do rotational physics with given damping
-	apply_physics( rotdamp, pi->desired_rotvel.xyz.x, pi->rotvel.xyz.x, sim_time, &new_vel.xyz.x, NULL );
-	apply_physics( rotdamp, pi->desired_rotvel.xyz.y, pi->rotvel.xyz.y, sim_time, &new_vel.xyz.y, NULL );
-	apply_physics( rotdamp, pi->desired_rotvel.xyz.z, pi->rotvel.xyz.z, sim_time, &new_vel.xyz.z, NULL );
-
-	Assert(is_valid_vec(&new_vel));
+	bool frit_ai_wants_to_move = Framerate_independent_turning && !IS_MAT_NULL(&pi->ai_desired_orient);
```

Comment:
What does "frit" in this variable stand for?  It's a long variable name, but I want to make sure that it's readable for everyone who needs to read this function.

Reply:
`Framerate_independent_turning `. I was hoping that would be more obvious, but I wanted to make sure in some way explicitly that the flag is bundled into the boolean, _as well_ as the ai 'wanting to move'. I'm not sure what a better name would be without making it cumbersomely long.

Reply:
I think it's a fairly good name, maybe just add a comment then.